pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -50,6 +51,19 @@
     )
 
 
+def isnotebook():
+    try:
+        shell = get_ipython().__class__.__name__
+        if shell == "ZMQInteractiveShell":
+            return True  # Jupyter notebook or qtconsole
+        elif shell == "TerminalInteractiveShell":
+            return False  # Terminal running IPython
+        else:
+            return False  # Other type (?)
```

Comment:
@maxcapodi78 I think we could simplify this section with:
```suggestion
        if shell == "ZMQInteractiveShell":
            return True  # Jupyter notebook or qtconsole
        else:
            return False
```